index.js

```diff
@@ -56,14 +58,34 @@ function groups (groups, data, aggregations, thisArg, additionalArgs) {
   data = Array.isArray(data) ? data : data.features
   var args = [undefined, undefined, undefined].concat(additionalArgs)
 
+  var tree = rbush(data.length)
```

Comment:
In two extremely poorly contained trials (during the first one I was actually using my laptop), it doesn't actually seem to be a big factor:

| Branching Factor | Trial 1 (using computer) | Trial 2 |
| --- | --- | --- |
| 100000 | 1m 42.8s | 1m 34.6s |
| 10000 | 1m 44.4s | 1m 32.7s |
| 1000 | 1m 35.7s | 1m 33.2s |
| 100 | 1m 39.8s | 1m 32.2s |

I'll leave as default for now and someone smarter can come along and optimize in the future